easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-foss-2020b.eb

```diff
@@ -0,0 +1,42 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# Based on STAR-2.7.7a-GCC-10.2.0.eb 
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'HH-suite'
+version = '3.3.0'
+
+homepage = 'https://github.com/soedinglab/hh-suite'
+description = """The HH-suite is an open-source software package 
+for sensitive protein sequence searching based on the pairwise 
+alignment of hidden Markov models (HMMs)."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

Comment:
Actually, no lapack or blas dependencies right? There seems to exist mpi variants though, so -> gompi?

Reply:
That worked. I think I was confused as there is already a `foss` version, albeit a bit older.

Reply:
"back in the day" we used to put everything into foss and intel. Bumping old stuff I always take a closer look to see if they were there unnecessarily.

Reply:
That does make sense. You are around longer than I am, so you know the history of EasyBuild better than I do.